reviewdog.go

```diff
@@ -25,6 +25,11 @@ func NewReviewdog(toolname string, p Parser, c CommentService, d DiffService) *R
 	return &Reviewdog{p: p, c: c, d: d, toolname: toolname}
 }
 
+func RunFromResult(ctx context.Context, c CommentService, results []*CheckResult,
```

Comment:
**[golint]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
exported function RunFromResult should have comment or be unexported